src/main/java/com/parasoft/report/transformer/XMLToSarif.java

```diff
@@ -96,8 +104,18 @@ private void checkInputAndOutputReportParams() {
     private void checkProjectRootPathsParam() {
         if (this.projectRootPaths != null && !this.projectRootPaths.trim().isEmpty()) {
             String[] pathsArray = this.projectRootPaths.trim().split(",");
-            // TODO: validate paths
-            this.projectRootPaths = String.join(",", Arrays.stream(pathsArray).map(String::trim).toArray(String[]::new));
+            String[] processedPaths = Arrays.stream(pathsArray).map((path) -> path.trim().replace("\\", "/")).toArray(String[]::new);
+            for (String path : processedPaths) {
+                if (!this.isAbsolutePath(path)) {
+                    throw new IllegalArgumentException(MessageFormat.format("Project root path must be an absolute path: {0}", path));
+                }
+                File rootFile = new File(path);
+                if (!rootFile.exists()) {
+                    Logger.warn(MessageFormat.format("WARN: Project root path does not exist: {0}.", path));
```

Comment:
Maybe we can make it more specific:
`WARN: Project root path does not exist on this machine: xxx`